Fix docstrings in `mlflow/tracing`

The following file contains incorrect docstrings. We want to fix them.

```
mlflow/tracing/export/inference_table.py:52:9: D417 Missing argument description in the docstring for `export`: `root_spans`
mlflow/tracing/export/mlflow.py:44:9: D417 Missing argument description in the docstring for `export`: `root_spans`
mlflow/tracing/provider.py:35:5: D417 Missing argument description in the docstring for `start_detached_span`: `experiment_id`
mlflow/tracing/utils.py:122:5: D417 Missing argument description in the docstring for `deduplicate_span_names_in_place`: `spans`
```

--- a/mlflow/tracing/export/inference_table.py
+++ b/mlflow/tracing/export/inference_table.py
@@ -55,8 +55,7 @@ class InferenceTableSpanExporter(SpanExporter):
 
         Args:
             root_spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
-                Only root spans for each trace are passed to this method. The root spans are
-                the spans that have no parent span.
+                Only root spans for each trace are passed to this method.
         """
         for span in root_spans:
             if span._parent is not None:

--- a/mlflow/tracing/export/inference_table.py
+++ b/mlflow/tracing/export/inference_table.py
@@ -54,8 +54,9 @@ class InferenceTableSpanExporter(SpanExporter):
         Export the spans to MLflow backend.
 
         Args:
-            spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
-                Only root spans for each trace are passed to this method.
+            root_spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
+                Only root spans for each trace are passed to this method. The root spans are
+                the spans that have no parent span.
         """
         for span in root_spans:
             if span._parent is not None:

--- a/mlflow/tracing/export/mlflow.py
+++ b/mlflow/tracing/export/mlflow.py
@@ -47,8 +47,7 @@ class MlflowSpanExporter(SpanExporter):
 
         Args:
             root_spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
-                Only root spans for each trace are passed to this method. The root spans are
-                the spans that have no parent span.
+                Only root spans for each trace are passed to this method.
         """
         for span in root_spans:
             if span._parent is not None:

--- a/mlflow/tracing/export/mlflow.py
+++ b/mlflow/tracing/export/mlflow.py
@@ -46,8 +46,9 @@ class MlflowSpanExporter(SpanExporter):
         Export the spans to MLflow backend.
 
         Args:
-            spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
-                Only root spans for each trace are passed to this method.
+            root_spans: A sequence of OpenTelemetry ReadableSpan objects to be exported.
+                Only root spans for each trace are passed to this method. The root spans are
+                the spans that have no parent span.
         """
         for span in root_spans:
             if span._parent is not None:

--- a/mlflow/tracing/provider.py
+++ b/mlflow/tracing/provider.py
@@ -43,6 +43,8 @@ def start_detached_span(
         name: The name of the span.
         parent: The parent OpenTelemetry span. If not provided, the span will be created as a root
                 span.
+        experiment_id: The ID of the experiment. This is used to associate the span with a specific
+                       experiment in MLflow.
 
     Returns:
         The newly created OpenTelemetry span.

--- a/mlflow/tracing/provider.py
+++ b/mlflow/tracing/provider.py
@@ -44,7 +44,7 @@ def start_detached_span(
         parent: The parent OpenTelemetry span. If not provided, the span will be created as a root
                 span.
         experiment_id: The ID of the experiment. This is used to associate the span with a specific
-                       experiment in MLflow.
+            experiment in MLflow.
 
     Returns:
         The newly created OpenTelemetry span.

--- a/mlflow/tracing/utils.py
+++ b/mlflow/tracing/utils.py
@@ -382,4 +382,3 @@ def exclude_immutable_tags(tags: Dict[str, str]) -> Dict[str, str]:
 
 def generate_request_id() -> str:
     return uuid.uuid4().hex
-           

--- a/mlflow/tracing/utils.py
+++ b/mlflow/tracing/utils.py
@@ -131,7 +131,7 @@ def deduplicate_span_names_in_place(spans: List[LiveSpan]):
         ["red", "red", "blue"] -> ["red_1", "red_2", "blue"]
 
     Args:
-        trace_data: The trace data object to deduplicate span names.
+        spans: A list of spans to deduplicate.
     """
     span_name_counter = Counter(span.name for span in spans)
     # Apply renaming only for duplicated spans
@@ -382,3 +382,4 @@ def exclude_immutable_tags(tags: Dict[str, str]) -> Dict[str, str]:
 
 def generate_request_id() -> str:
     return uuid.uuid4().hex
+           

